.github/workflows/acp-test.yaml

```diff
@@ -0,0 +1,17 @@
+name: acp chart test
```

Comment:
Would it be possible to make it more generic, so we can share some steps between these workflows?

Reply:
I am not sure if that would be possible: https://github.community/t/sharing-steps-between-jobs-and-jobs-between-workflows/17751/6